### PR TITLE
manual pagination

### LINKED ISSUE #23: Paginate comments
- remove symfony paginator bundle
- redo pagination by hand via doctrine paginator
- adjust template

### PR DESCRIPTION
**Pull request by ThomasLdev**
closes #23

### CODE REVIEW

File: src/Controller/TrickController.php [hunk 1]
```diff
@@ -107,23 +108,22 @@ public function edit(Request $request, Trick $trick, UploaderHelper $uploaderHel
...
-        /*$queryBuilder = $commentRepository->getCommentsQueryBuilder($trick);
-        $pagination = $paginator->paginate(
-            $queryBuilder,/
-            $request->query->getInt('page', 1),
-            5
-        );*/
+        $limit = 5;
+        $page = (int)$request->query->get("page", 1);
+        $comments = $commentRepository->getPaginatedComments($page, $limit, $trick);
+        $pagination = array(
```
Comment (crazymad) [inline src/Controller/TrickController.php:115]: [] instead of array

File: src/Controller/TrickController.php [hunk 2]
```diff
@@ -107,23 +108,22 @@ public function edit(Request $request, Trick $trick, UploaderHelper $uploaderHel
...
-            'trick' => $trick
-        ]);
 
-        /*$queryBuilder = $commentRepository->getCommentsQueryBuilder($trick);
-        $pagination = $paginator->paginate(
-            $queryBuilder,/
-            $request->query->getInt('page', 1),
-            5
-        );*/
+        $limit = 5;
```
Comment (crazymad) [inline src/Controller/TrickController.php:112]: CONST

Comment (ThomasLdev) [reply src/Controller/TrickController.php:112]: replaced by : define("App\Controller\limit", 5);

File: src/Controller/TrickController.php (hunk 1 above)
Comment (ThomasLdev) [reply src/Controller/TrickController.php:115]: Done

**Pull request merged**